tests/VersionTest.php

```diff
@@ -14,11 +17,12 @@ public function test_it_can_be_initialized()
         $this->assertInstanceOf(SemVer\Version::class, $this->version);
     }
 
+    /**
+     * @expectedException PHLAK\SemVer\Exceptions\InvalidVersionException
+     */
```

Comment:
I'd like to use the PHPUnit methods instead of docblock annotations here. Can we use the following in the test method code instead?

```php
$this->expectException(SemVer\Exceptions\InvalidVersionException::class);
```

Reply:
Yes. You're right. I use the `expected exception annotation` because I want be compatible with `^5.7`. Once we define `^6.5 || ^7.0`, this will be unnecessary.